.github/workflows/maven.yml

```diff
@@ -29,29 +28,37 @@ jobs:
       with:
         servers: '[{"id": "github", "username": "${{ github.actor }}", "password": "${{ github.token}}"}, {"id": "github-annotation", "username": "${{ github.actor }}", "password": "${{ github.token }}"}]'
         repositories: '[{"id" : "github", "url" : "https://maven.pkg.github.com/Ubique-OSS/springboot-swagger3"}, {"id" : "github-annotation", "url" : "https://maven.pkg.github.com/Ubique-OSS/springboot-swagger3-annotations"}]'
-    - name: Build with Maven
+    
+    - name: Cache Maven packages
```

Comment:
Just curious: I tried that with one of my projects, and it failed to correctly cache the java packages. Does it work here?

Reply:
Interesting. We'll find out :)

Reply:
What exactly was the problem you were facing?

Reply:
Maven continued to download all packages every time the workflow started. So I think I did something wrong, but after 10 runs I gave up ;) It was for 

https://github.com/c4dt/notifyme-sdk-backend-epfl/blob/main/.github/workflows/build.yml#L24